aggregator/src/aggregator/http_handlers.rs

```diff
@@ -266,6 +267,32 @@ where
     }
 }
 
+/// A Trillium handler that returns an empty body with retry-after and location headers.
+#[derive(Clone)]
+struct EmptyBody {
+    location: String,
+}
+
+impl EmptyBody {
+    /// Return an EmptyBody with the location set to the relative path to the
+    /// aggregation job for the task.
+    fn for_task_and_job(task_id: &TaskId, aggregation_job_id: &AggregationJobId) -> Self {
+        Self {
+            location: format!("/tasks/{task_id}/aggregation_jobs/{aggregation_job_id}?step=0"),
```

Comment:
Hmm, is it right to hardcode step=0? If we were running a VDAF with more than one preparation step, then you'd eventually need to go to step=1. In practice, we only actually run Prio3 VDAFs, which finish at step 1 and thus will never have a query besides step=0, but I think Janus supports VDAFs with more steps than that and it'd be nice to maintain that. For instance, I think we have some tests that instantiate the dummy VDAF with multiple rounds so that we can exercise AggregationJobContinue.

Reply:
We could change this signature to `fn for_aggregation_job(task_id: &TaskId, aggregation_job_id: &AggregationJobId, step: u16) -> Self`. Note that we'll have to create a new enum for the return value of `handle_aggregate_continue()` to pass the aggregation job step out (instead of just `None`). The test expectations in `aggregation_job_continue.rs` will have to be updated as well.